src/reducers/browse.test.js

```diff
@@ -123,4 +123,13 @@ describe('browse reducer', () => {
 
     expect(browseReducer(state, action)).toEqual(state);
   });
+
+  it(`should handle ${actionTypes.CHANGE_BROWSE_LEVEL}`, () => {
+    const state = fromJS({
+      currentBrowseLevel: 'schools',
+    });
+    const action = actionCreators.changeBrowseLevel('schools');
```

Comment:
Let's actually change the browse level to test that the action works - otherwise we're not really testing anything!

Reply:
Done